app/controllers/ProjectController.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace GanttDashboard\App\Controllers;
+
+use Phalcon\Mvc\Controller;
+use GanttDashboard\App\Services\Project as ProjectService;
+use GanttDashboard\App\Models\Projects;
+use Phalcon\Http\ResponseInterface;
+
+class ProjectController extends Controller
+{
+    /**
+     * @var ProjectService
+     */
+    private $projectService;
+
+    /**
+     * Initializes the project service
+     * @return void
+     */
+    public function onConstruct(): void
+    {
+        $this->projectService = $this->getDi()->get(ProjectService::class);
+    }
+
+    /**
+     * If admin is logged in, registers new project via project service.
+     * @return ResponseInterface
+     */
+    public function registerAction(): ResponseInterface
+    {
+        $project = $this->request->getPost();
+        $errors = $this->projectService->register(new Projects(), $project);
+
+        if (0 == $errors->count()) {
+            $this->flashSession->success('Project registration successful');
+            $this->view->disable();
+            
+            return $this->response->redirect(['for' => 'registerProject']);
+        }
+
+        $this->view->setVar('errors', $errors);
+        $view = $this->view->render('Project', 'register');
```

Comment:
lowercase "P" - this resolved to a directory. UNIX will have unexpected behaviour for this